### PR TITLE
Remove Jenkins and add Linux support

### PR DESCRIPTION
**Pull request by fashaikh**

### CODE REVIEW

File: SimpleWAWS/Code/CsmExtensions/CsmManager.cs [hunk 1]
```diff
@@ -43,9 +44,9 @@ static CsmManager()
                 // Load all subscriptions
                 var csmSubscriptions = await CsmManager.GetSubscriptionNamesToIdMap();
                 _subscriptions = (SimpleSettings.Subscriptions.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
-                    .Concat(SimpleSettings.JenkinsSubscriptions.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)))
+                    .Concat(SimpleSettings.LinuxSubscriptions.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)))
                     //It can be either a displayName or a subscriptionId
-                    //For Jenkins it needs to be subscriptionId
+                    //For Linuxit needs to be subscriptionId
```
Comment (ahmelsayed) [inline SimpleWAWS/Code/CsmExtensions/CsmManager.cs]: spelling

File: SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs [hunk 2]
```diff
@@ -228,6 +216,21 @@ public static partial class CsmManager
 
                 await InitFunctionsContainer(resourceGroup);
             }
+            if (resourceGroup.SubscriptionType == SubscriptionType.Linux)
```
Comment (ahmelsayed) [inline SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs]: make it else if

File: SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs [hunk 3]
```diff
@@ -228,6 +216,21 @@ public static partial class CsmManager
...
+            {
+                // If the resourceGroup is assigned, don't mess with it
+                if (!string.IsNullOrEmpty(resourceGroup.UserId)) return resourceGroup;
```
Comment (ahmelsayed) [inline SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs]: add { }

File: SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs [hunk 4]
```diff
@@ -228,6 +216,21 @@ public static partial class CsmManager
...
+
+                var createdSites = new List<Task<Site>>();
```
Comment (ahmelsayed) [inline SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs]: You probably don't need a list for this. There is only 1 resource you're creating, right?

Comment (fashaikh) [inline SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs]: You  are right . I'll update this.

**Pull request merged**